src/hotspot/share/gc/g1/g1OopClosures.hpp

```diff
@@ -202,13 +202,20 @@ class G1RootRegionScanClosure : public ClaimMetadataVisitingOopIterateClosure {
 class G1ConcurrentRefineOopClosure: public BasicOopIterateClosure {
   G1CollectedHeap* _g1h;
   uint _worker_id;
+  bool _has_to_cset_ref;
```

Comment:
Similar to `_cards_refer_to_cset` , do you mind renaming `_has_to_cset_ref`  and `_has_to_old_ref`  to `_has_ref_to_cset` and `_has_ref_to_old`